sdk/identity/azure-identity-broker/azure/identity/broker/_browser.py

```diff
@@ -17,7 +17,13 @@
 
 
 class PopTokenRequestOptions(TokenRequestOptions):
-    """Options to use for pop token requests."""
+    """Options to use for pop token requests.
+
+    pop = None means non-PoP token request.
+    pop = True means mTLS PoP token request
+    pop = a Dict means SHR PoP token request
+
+    """
 
     pop: Union[bool, Mapping[str, str]]
```

Comment:
I'd suggest adding the value specifics to the `pop` docstring.
```suggestion
    pop: Union[bool, Mapping[str, str]]
    """PoP token request options.

    - If not specified or False, a non-PoP token request is made.
    - If True, an mTLS PoP token request is made.
    - If a dict, a Signed HTTP Request (SHR) PoP token request is made. The dict
      must contain the "resource_request_method", "resource_request_url", and "nonce" keys.
    """
```